lib/bundle.js

```diff
@@ -186,13 +189,19 @@ exports.bundle = function(moduleExpression, fileName, opts) {
         return path.resolve(config.pjson.baseURL, file);
       }), function invalidate(invalidated) {
         systemBuilder.invalidate(toFileURL(invalidated));
-      }, function rebuild() {
-        return bundle();
+      }, function rebuild(triggerFile) {
+        if (triggerFile != fileName &&
+            triggerFile.indexOf('jspm.browser.js') == -1) {
```

Comment:
Because the rebuild may be triggered by a file change that happened while running a build, we don't necessarily only have one triggerFile. So ideally we can forgo this argument.